### ISSUE TITLE
Yolov8 cant open/read images

### ISSUE THREAD

**Issue by AikynJ** [ultralytics/ultralytics#4006]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

train: Scanning C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\labels\train.cache... 48 images, 14 backgrounds, 0 corrupt: 100%|██████████| 48/48 [00:00<?, ?it/s]
[ WARN:0@20.427] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\211(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@17.857] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\23(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@15.231] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\45(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@12.616] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\26(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@10.098] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\19(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@7.585] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\22(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@5.036] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\21(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@20.429] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\29(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@17.859] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\29(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@15.233] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\13(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@12.617] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\36(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@10.100] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\25(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@7.587] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\36(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@5.038] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\13(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@2.452] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\46(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@2.453] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\32(2).jpg'): can't open/read file: check file path/integrity
val: Scanning C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\labels\train.cache... 48 images, 14 backgrounds, 0 corrupt: 100%|██████████| 48/48 [00:00<?, ?it/s]
[ WARN:0@40.764] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@38.239] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@35.710] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@33.188] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@30.596] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@28.073] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@25.554] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@20.383] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@23.039] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@17.836] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@15.321] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@12.761] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@10.192] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@7.595] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@4.992] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@40.787] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@38.262] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@35.732] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@33.210] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@30.617] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@28.095] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@25.575] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@23.058] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@20.403] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@12.768] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@15.332] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@17.855] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@10.195] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@7.597] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@4.994] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\1(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@2.423] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
[ WARN:0@2.425] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\16(2).jpg'): can't open/read file: check file path/integrity
Plotting labels to runs\detect\train6\labels.jpg... 
optimizer: AdamW(lr=0.002, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0)
Image sizes 640 train, 640 val
Using 8 dataloader workers
Logging results to runs\detect\train6
Starting training for 3 epochs...

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size
  0%|          | 0/3 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "C:\Users\Admin\OneDrive\Рабочий стол\new\main.py", line 12, in <module>
    train_model()
  File "C:\Users\Admin\OneDrive\Рабочий стол\new\main.py", line 8, in train_model
    model.train(data="config1.yaml", epochs=3)  # train the model
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\engine\model.py", line 372, in train
    self.trainer.train()
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\engine\trainer.py", line 192, in train
[ WARN:0@63.482] global loadsave.cpp:248 cv::findDecoder imread_('C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\28(2).jpg'): can't open/read file: check file path/integrity
    self._do_train(world_size)
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\engine\trainer.py", line 315, in _do_train
    for i, batch in pbar:
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\tqdm\std.py", line 1178, in __iter__
    for obj in iterable:
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\data\build.py", line 38, in __iter__
    yield next(self.iterator)
          ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\dataloader.py", line 633, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\dataloader.py", line 1345, in _next_data
    return self._process_data(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\dataloader.py", line 1371, in _process_data
    data.reraise()
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\_utils.py", line 644, in reraise
    raise exception
FileNotFoundError: Caught FileNotFoundError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\_utils\worker.py", line 308, in _worker_loop
    data = fetcher.fetch(index)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\_utils\fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\torch\utils\data\_utils\fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\data\base.py", line 240, in __getitem__
    return self.transforms(self.get_image_and_label(index))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\data\base.py", line 246, in get_image_and_label
    label['img'], label['ori_shape'], label['resized_shape'] = self.load_image(index)
                                                               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ultralytics\data\base.py", line 153, in load_image
    raise FileNotFoundError(f'Image Not Found {f}')
FileNotFoundError: Image Not Found C:\Users\Admin\OneDrive\Рабочий стол\realdataset3\images\train\211(2).jpg


### Environment

Ultralytics YOLOv8.0.139  Python-3.11.4 torch-2.0.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3050 Ti Laptop GPU, 4096MiB)

### Minimal Reproducible Example

```python
from ultralytics import YOLO

def train_model():
    # Load a model
    model = YOLO("yolov8n.yaml")  # build a new model from scratch

    # Use the model
    model.train(data="config1.yaml", epochs=1)  # train the model

if __name__ == '__main__':
    # Call the training function inside the if __name__ == '__main__': block
    train_model()
```



### Additional

This is config1.yaml:

```python
path: 'C:/Users/Admin/OneDrive/Рабочий стол/new/realdataset3'
train: images/train
val: images/train

names:
  0: stamp
```

  
So, basically as I understood, my programme can't read the files from path 



### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@AikynJ 👋 hi, thanks for letting us know about this possible problem with YOLO 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub with `git pull` or `pip install -U ultralytics` to ensure your problem has not already been solved in the latest code version.
- ✅ **Unmodified** – Your problem must be reproducible using official Ultrlaytics code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/ultralytics/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by glenn-jocher** [comment]
@AikynJ hello, 

Thank you for reaching out. To better assist you with the inquiry, we ask you to provide more context for your question or the situation you are facing with the YOLOv8 repo. 

For instance, are you having a specific issue with functionality, encountering bugs, or do you need assistance with implementation? Please provide as much detail as possible so we can support you effectively. 

Looking forward to your response!

**Comment by AikynJ** [comment]
@glenn-jocher Hello,

Basically when I try to train YOLOv8 on my custom dataset, it can't read/open images.

**Comment by glenn-jocher** [comment]
@AikynJ hello, 

It seems like the model is having trouble accessing your images during training. This error typically arises from issues with the file paths specified in your dataset configuration. It's worth double-checking that the paths for your image files are correctly specified, relative to the working directory. 

Also, ensure the images are in a format that YOLOv8 can interpret (such as .jpg or .png) and that they're not corrupted. 

I hope this helps! Let me know how you get on or if there's anything else you're unsure about.

**Comment by hitbuyi** [comment]
I have same issue

**Comment by glenn-jocher** [comment]
@hitbuyi @AikynJ this appears to be a file path issue. The most likely causes are:

1. **Cyrillic characters in path**: The path contains "Рабочий стол" which can cause OpenCV file reading issues on Windows
2. **Path resolution mismatch**: Your config shows `/new/` in the path but the error paths don't include it

Try moving your dataset to a simple path without special characters like `C:\dataset\realdataset3` and update your config accordingly. This should resolve the file access errors you're seeing.

**Comment by andife** [comment]
I may have a similar behavior, in combination with Clearml. During training, the data can suddenly no longer be found. Any idea?

```
                   all         24         19      0.784          1      0.879      0.549      0.742      0.947      0.821      0.624
      Epoch    GPU_mem   box_loss   seg_loss   cls_loss   dfl_loss  Instances       Size
      28/50      1.43G     0.3575     0.4109      1.471     0.2383          1        640: 100%|██████████| 8/8 [00:00<00:00, 27.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  67%|██████▋   | 4/6 [00:00<00:00, 43.97it/s]
Traceback (most recent call last):
  File "/home/data/clearml/venvs-builds/3.11/task_repository/ClearML_Onboarding/training/src/pipeline/training_pipeline_tls_base.py", line 316, in <module>
    main(input_dir=input_dir, output_dir=output_dir, config_path=config_path, config=config_file)
  File "/home/data/clearml/venvs-builds/3.11/task_repository/ClearML_Onboarding/training/src/pipeline/training_pipeline_tls_base.py", line 260, in main
    training_pipeline.run()
  File "/home/data/clearml/venvs-builds/3.11/task_repository/ClearML_Onboarding/training/src/pipeline/training_pipeline_tls_base.py", line 156, in run
    metrics = yolo_pipeline.run(self.__config)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/task_repository/ClearML_Onboarding/training/../training/src/pipeline/yolo_pipeline.py", line 73, in run
    model.train(
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/engine/model.py", line 810, in train
    self.trainer.train()
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/engine/trainer.py", line 208, in train
    self._do_train(world_size)
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/engine/trainer.py", line 433, in _do_train
    self.metrics, self.fitness = self.validate()
                                 ^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/engine/trainer.py", line 617, in validate
    metrics = self.validator(self)
              ^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/engine/validator.py", line 173, in __call__
    for batch_i, batch in enumerate(bar):
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/data/build.py", line 48, in __iter__
    yield next(self.iterator)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 708, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 1480, in _next_data
    return self._process_data(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 1505, in _process_data
    data.reraise()
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/_utils.py", line 733, in reraise
    raise exception
FileNotFoundError: Caught FileNotFoundError in DataLoader worker process 6.
Original Traceback (most recent call last):
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/_utils/worker.py", line 349, in _worker_loop
    data = fetcher.fetch(index)  # type: ignore[possibly-undefined]
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 52, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/data/base.py", line 288, in __getitem__
    return self.transforms(self.get_image_and_label(index))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/data/base.py", line 294, in get_image_and_label
    label["img"], label["ori_shape"], label["resized_shape"] = self.load_image(index)
                                                               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/data/clearml/venvs-builds/3.11/lib/python3.11/site-packages/ultralytics/data/base.py", line 165, in load_image
    raise FileNotFoundError(f"Image Not Found {f}")
FileNotFoundError: Image Not Found /home/data/clearml/cache/storage_manager/datasets/ds_36df3ddbacd44436aa0e8934c45e092f/val/images/FinWpiece_32.png
```

**Comment by glenn-jocher** [comment]
@andife your ClearML issue appears to be different - it looks like the dataset cache path is changing during training, possibly due to ClearML's dataset management. Try setting a fixed local dataset path in your YAML config rather than relying on ClearML's cache paths, or ensure your dataset is properly uploaded and accessible throughout the training process.

For the original issue, the file path with Cyrillic characters is definitely the root cause - OpenCV has known issues with non-ASCII characters in Windows file paths.
